### ISSUE TITLE
Optimize the search for inferencing

### ISSUE THREAD

**Issue by ghost** [DrSleep/nas-segm-pytorch#7]
@DrSleep Thank you for your hard work,

My goal is to search for an architecture that balance accuracy and high inferencing speed on cpu.
- Is there an option to limit the search space to not exceed a certain limit of `parameters` `flops` `adds`?
- How to inference an image using cpu only?

**Comment by DrSleep** [comment]
1. There is no such option in this repository other than by trial and error: try changing different hyperparameters to see how the generated architectures fit into your requirements. In particular, you can control how many blocks are in the decoder via [DEC_NUM_CELLS](https://github.com/DrSleep/nas-segm-pytorch/blob/master/src/utils/default_args.py#L70) and the number of layers in the cells via [CELL_NUM_LAYERS](https://github.com/DrSleep/nas-segm-pytorch/blob/master/src/utils/default_args.py#L65)

2. For the cpu inference, either remove all `.cuda()` calls or replace them with the `.cpu()` calls instead. For more, refer to the pytorch documentation.

**Comment by deepseek** [comment]
Thanks
